### DISCUSSION TITLE
캐릭터의 체형을 Default 체형으로 변경할 수 있나요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#664] [5. Q&A]
월드에 들어오는 순간 캐릭터 체형을 Default로 변경이 가능할까요?

목적은 너무 작거나 너무 큰 체형 같은 것을 기본 체형으로 변경하고 싶습니다.

**Comment by yoonyoungz** [comment]
체형 템플릿을 default로 적용하는것은 어려운 걸로 알고 있어요.

**Comment by hhnmsLeesan** [comment]
제페토 캐릭터에 붙어있는 Zepeto Context 라는 컴포넌트를 비활성화하면 체형이 기본체형으로 돌아옵니다.
